README.md

```diff
@@ -22,11 +22,11 @@ SparseArrays.jl provides functionality for working with sparse arrays in Julia.
 
 ## Using newer version of this package with julia master
 
-You need to build Julia from scratch to use the git version (or other version) of this package. The build process is the same, but `DEPS_GIT` should be set when building i.e. `make DEPS_GIT="SparseArrays" ...`. The other option is to manually select the commit in `stdlib/SparseArrays.version`. 
+To use a newer version of this package, you need to build Julia from scratch. The build process is the same as any other build except that `DEPS_GIT="SparseArrays"` should be passed to `make` i.e. `make DEPS_GIT="SparseArrays"`. Then after the build is complete, the git repo in `stdlib/SparseArrays` can be used to check out the desired commit/branch. Alternatively, you need to change the commit used in `stdlib/SparseArrays.version`. The first method does not require rebuilding julia to update the library.
```

Comment:
What do you mean with 
> The first method does not require rebuilding julia to update the library.

What is the first method referring to?

Reply:
using `DEPS_GIT`

Reply:
Why doesn't that require a rebuild?

Reply:
because if you can checkout the git repo.

Reply:
i guess there is no need to rebuild it in both cases but it's more convenient in the latter case? (tho i haven't tested it)

Reply:
Yea okay, but only because SparseArrays is not currently in the sysimage.

Reply:
should i add that detail?

Reply:
if it was in the sysimage, how would the git route look like?

Reply:
Same, but you would always have to rebuild.

Reply:
I think we can merge this for now - but if we put SparseArrays back in the sysimage, then we'll need to update.